wallets/provider-ledger/package.json

```diff
@@ -14,28 +14,30 @@
     "src"
   ],
   "scripts": {
-    "build": "node ../../scripts/build/command.mjs --path wallets/provider-ledger",
+    "build": "node ../../scripts/build/command.mjs --path wallets/provider-ledger --external-all-except @ledgerhq/errors,@ledgerhq/hw-app-eth,@ledgerhq/hw-app-solana,@ledgerhq/hw-transport-webhid,@ledgerhq/types-cryptoassets,@ledgerhq/types-devices,",
     "ts-check": "tsc --declaration --emitDeclarationOnly -p ./tsconfig.json",
     "clean": "rimraf dist",
     "format": "prettier --write '{.,src}/**/*.{ts,tsx}'",
     "lint": "eslint \"**/*.{ts,tsx}\" --ignore-path ../../.eslintignore"
   },
   "dependencies": {
-    "@ledgerhq/errors": "^6.16.4",
-    "@ledgerhq/hw-app-eth": "^6.36.0",
-    "@ledgerhq/hw-app-solana": "^7.1.6",
-    "@ledgerhq/hw-transport-webhid": "^6.28.6",
-    "@ledgerhq/types-cryptoassets": "^7.11.0",
-    "@ledgerhq/types-devices": "^6.24.0",
     "@rango-dev/signer-solana": "^0.31.1-next.1",
     "@rango-dev/wallets-shared": "^0.36.1-next.1",
     "@solana/web3.js": "^1.91.4",
     "@types/w3c-web-hid": "^1.0.2",
     "bs58": "^5.0.0",
-    "ethers": "^6.11.1",
+    "ethers": "^6.13.2",
     "rango-types": "^0.1.69"
   },
+  "devDependencies": {
+    "@ledgerhq/errors": "^6.16.4",
+    "@ledgerhq/hw-app-eth": "^6.36.0",
+    "@ledgerhq/hw-app-solana": "^7.1.6",
+    "@ledgerhq/hw-transport-webhid": "^6.28.6",
+    "@ledgerhq/types-cryptoassets": "^7.11.0",
+    "@ledgerhq/types-devices": "^6.24.0"
```

Comment:
@RyukTheCoder I'm not sure about this change. Please double check with @yeager-eren if you didn't already.

Reply:
moving to `devDependencies` will break our `tsc` users. please revert them back.

Reply:
reverted moving ledgerhq to devDependencies.
[fix: revert moving ledgerhq to dev dependencies](https://github.com/rango-exchange/rango-client/pull/842/commits/788ae2356cdf505c17bd6a469054c7863054fd7f)